File: py110-intermediate/exercises/easy_5/delete_vowels.py
def remove_vowels(lst):
    new_lst = []
    for elem in lst:
        new_str = ''
        for char in elem:
            if char not in 'aeiouAEIOU':
                new_str += char
        new_lst.append(new_str)

    return new_lst

# All of these examples should print True
original = ['abcdefghijklmnopqrstuvwxyz']
expected = ['bcdfghjklmnpqrstvwxyz']
print(remove_vowels(original) == expected)        # True

original = ['green', 'YELLOW', 'black', 'white']
expected = ['grn', 'YLLW', 'blck', 'wht']
print(remove_vowels(original) == expected)        # True

original = ['ABC', 'AEIOU', 'XYZ']
expected = ['BC', '', 'XYZ']
print(remove_vowels(original) == expected)        # True

File: js210/exercises/stringProcessing/deleteVowels.js
function removeVowels(strings) {
  const vowelsRemoved = [];

  for (let word of strings) {
    let newWord = '';
    for (let char of word) {
      if (!/[aeiouAEIOU]/.test(char)) {
        newWord += char;
      }
    }

    vowelsRemoved.push(newWord);
  }

  return vowelsRemoved;
}

console.log(removeVowels(['abcdefghijklmnopqrstuvwxyz']));         // ["bcdfghjklmnpqrstvwxyz"]
console.log(removeVowels(['green', 'YELLOW', 'black', 'white']));  // ["grn", "YLLW", "blck", "wht"]
console.log(removeVowels(['ABC', 'AEIOU', 'XYZ']));                // ["BC", "", "XYZ"]